releasenotes/notes/40339.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 40339
+
+releaseNotes:
+- |
+  **Added** `istioctl x uninstall` promoted to `istioctl uninstall`.
```

Comment:
**Promoted** is a keyword.

```suggestion
  **Promoted** `istioctl experimental uninstall` to `istioctl uninstall`.
```

Reply:
will change. thanks @ericvn

Reply:
@ericvn done with the 2nd commit. Thanks.